engine/src/exec-pasteboard.cpp

```diff
@@ -279,7 +279,7 @@ MCTransferType MCPasteboardFindTransferTypeForLegacyClipboard(const MCClipboard*
     // passed that the behaviour can now be corrected but, until then, it is
     // disabled and the "text over images" method is used instead.
     //
-    if (false)
+    if (/* DISABLES CODE */ (false))
```

Comment:
Is there a good reason to use `if (false)` rather than just `ifdef`fing the block with something undefined?